convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -170,7 +171,7 @@ def run(self):
             logger.info(message)
             self.add_message(
                 level="INFO",
-                id="SPECIAL_PACKAGES_REMOVED",
+                id="SPECIAL_PACKAGES_TO_REMOVE",
```

Comment:
Breaking change to change ID
```suggestion
                id="SPECIAL_PACKAGES_TO_REMOVED",
```

Reply:
https://github.com/oamg/convert2rhel/pull/1447